### PR TITLE
New edition of This Week in Solid - now This Month in Solid

### PR DESCRIPTION
**Pull request by Vinnl**
The cadence has been changed around a bit, so the newsletter will be monthly now.

### CODE REVIEW

**Comment by RubenVerborgh** [comment]
Hi all,

I'm sorry but it's too complicated for me to review the newsletter when the actual PR that I need to approve contains 22 commits and 50 files.. We really need to find a system where it is just the newsletter. Happy to help when we've found that.

Suggestion: have a branch on which the newsletter is prepared.

Thanks for your understanding,

Ruben

**Comment by Vinnl** [comment]
@RubenVerborgh I don't know what to do here; I submitted a PR to arrange exactly that to the process repo (https://github.com/solid/process/pull/195), but I can't approve that myself obviously. But if you want to just look at the next newsletter, see [_posts/newsletter/2020-04-09-this-month-in-solid.md](https://github.com/solid/solidproject.org/pull/185/files#diff-5559449ac4f50d5ef2e57d2839d5c8fa).

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+layout: newsletter
+title: This Month in Solid 2020-04-09
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:3]: Update date?

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+---
+{% include twis-intro.md %}
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Travis Vachon, David Faveris, and Astrid Gamoneda for presenting at the first Solid World last Thursday.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:16]: last Thursday might need adjustment (unless the newsletter will be backdated)

Comment (RubenVerborgh) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md:16]: Is there a recording?

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+In the third talk, [Astrid Gamoneda Arrunada](https://github.com/empathyco/solid-pods-management/commits?author=astrd) from Spain presented a Solid app called [OhMyPod!](https://ohmypod.netlify.com/login)  developed by [Empathy](https://www.empathy.co/). OhMyPod! has a file management system with the ability to add friends with personal profiles. OhMyPod! is built using Inrupt React Generator, LDflex library and solid-file-client library. Astrid also looked at rdflib, which is powerful but complicated and somewhat slow. 
+
+If you missed Solid World you can [watch the recording [here](https://vimeo.com/404963568). 
+
+The [next Solid World](https://www.eventbrite.com/e/solid-world-tickets-101812024506?aff=erelexpmlt) will happen soon on the 7th May 2020. If you have any content requests or would like to present at the next Solid world email mitzil@inrupt.com. 
+
+### In Other News  
+
+Darcy, a Solid based app led by [Christian Buggedei](https://github.com/JollyOrc) [won a €10k seed award from Mozilla](https://www.patreon.com/posts/35809190) and will be speaking at [Solid World May](https://www.eventbrite.com/e/solid-world-tickets-101812024506).
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:30]: what does the app do?

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+## Updates
+
+### [Events](https://solidproject.org/events)
+
+Solid World is an opportunity to meet people who are interested in or currently working on Solid.
+
+A big thank you to Travis Vachon, David Faveris, and Astrid Gamoneda for presenting at the first Solid World last Thursday. 
+
+In the first talk, [Travis Vachon](https://github.com/travis) from the USA presented a Solid app called [Concept](https://github.com/travis/concept). Concept is a Google docs equivalent that allows users to control their data. Travis built the concept with the aim to allow grassroots campaigns to organise themselves. Patch and WebSockets allowed for rudimentary collaborative editing and now wants to focus on embedding calendars and spreadsheets.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:18]: Google Docs (cap)

Comment (RubenVerborgh) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md:18]: *he* wants

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+In the second talk, [David Faveris](https://forum.solidproject.org/u/Smag0) from France presented a Solid app called [Solidarity](https://github.com/scenaristeur/solidarity). Solidarity is a chat app built using the Shighl library, which fills the gap between the Solidarity UI and Solid Pods. David worked closely with designer Julian Cole who has been working with Shighl to allow for easy installation and minimal dependencies while working with linked data. 
+
+In the third talk, [Astrid Gamoneda Arrunada](https://github.com/empathyco/solid-pods-management/commits?author=astrd) from Spain presented a Solid app called [OhMyPod!](https://ohmypod.netlify.com/login)  developed by [Empathy](https://www.empathy.co/). OhMyPod! has a file management system with the ability to add friends with personal profiles. OhMyPod! is built using Inrupt React Generator, LDflex library and solid-file-client library. Astrid also looked at rdflib, which is powerful but complicated and somewhat slow.
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:22]: Might want to phrase that last bit as "she found…"

File: _posts/newsletter/2020-04-09-this-month-in-solid.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+In the second talk, [David Faveris](https://forum.solidproject.org/u/Smag0) from France presented a Solid app called [Solidarity](https://github.com/scenaristeur/solidarity). Solidarity is a chat app built using the Shighl library, which fills the gap between the Solidarity UI and Solid Pods. David worked closely with designer Julian Cole who has been working with Shighl to allow for easy installation and minimal dependencies while working with linked data. 
+
+In the third talk, [Astrid Gamoneda Arrunada](https://github.com/empathyco/solid-pods-management/commits?author=astrd) from Spain presented a Solid app called [OhMyPod!](https://ohmypod.netlify.com/login)  developed by [Empathy](https://www.empathy.co/). OhMyPod! has a file management system with the ability to add friends with personal profiles. OhMyPod! is built using Inrupt React Generator, LDflex library and solid-file-client library. Astrid also looked at rdflib, which is powerful but complicated and somewhat slow. 
+
+If you missed Solid World you can [watch the recording [here](https://vimeo.com/404963568).
```
Comment (RubenVerborgh) [inline _posts/newsletter/2020-04-09-this-month-in-solid.md:24]: extra `[`

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 1 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md:3]: It was a bit delayed due to having to make sure all links kept working correctly with the switch to a monthly newsletter, and Inrupt having an All-hands, but the content covers the period up to that date, so we're backdating it indeed.

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 2 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md:16]: Yep, see line 24. (And as mentioned in the other comment, indeed we're backdating.)

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 6 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md]: :+1: fixed.

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 5 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md]: :+1: fixed.

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 4 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md]: :+1: fixed.

File: _posts/newsletter/2020-04-09-this-month-in-solid.md (hunk 3 above)
Comment (Vinnl) [reply _posts/newsletter/2020-04-09-this-month-in-solid.md]: I added a blurb from their Twitter account in 9dac4dce30b855f8b2756273b7a3fa9e0c160b22.

File: _posts/newsletter/next.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+In the third talk, [Astrid Gamoneda Arrunada](https://github.com/empathyco/solid-pods-management/commits?author=astrd) from Spain presented a Solid app called [OhMyPod!](https://ohmypod.netlify.com/login)  developed by [Empathy](https://www.empathy.co/). OhMyPod! has a file management system with the ability to add friends with personal profiles. OhMyPod! is built using Inrupt React Generator, LDflex library and solid-file-client library. Astrid also looked at rdflib, which is powerful but complicated and somewhat slow. 
+
+If you missed Solid World you can [watch the recording [here](https://vimeo.com/404963568). 
+
+The [next Solid World](https://www.eventbrite.com/e/solid-world-tickets-101812024506?aff=erelexpmlt) will happen soon on the 7th May 2020. If you have any content requests or would like to present at the next Solid world email mitzil@inrupt.com. 
+
+### In Other News  
+
+Darcy, a Solid based app led by [Christian Buggedei](https://github.com/JollyOrc) [won a €10k seed award from Mozilla](https://www.patreon.com/posts/35809190) and will be speaking at [Solid World May](https://www.eventbrite.com/e/solid-world-tickets-101812024506).
```
Comment (RubenVerborgh) [inline _posts/newsletter/next.md:30]: missing `he` [will be speaking at]

Comment (Vinnl) [reply _posts/newsletter/next.md]: Fixed before merging.

**Comment by timbl** [comment]
In fact he spoke at it

**Pull request merged**